docs/book/releases/v2.2.0.md

```diff
@@ -0,0 +1,68 @@
+  - Fixed recreating static file share volume with the same PV name.
+  - Fixed metadata syncer to disallow deleting volume physically from the datastore when PV is deleted by user.
+  - Fixed VCP to CSI Migration to support datastore path in the volume handle. Refer [issue-628](https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/628#issuecomment-771258857) for more detail.
+  - Fixed CSI full sync not registering volumes and was failing with `Duplicated entity for each entity type in one cluster`.
+  - Fixed volume provisioning when empty datacenter is present in the vCenter inventory.
+
+## Deployment files
+
+- https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/v2.2.0
+
+## Kubernetes Release
+
+- Minimum: 1.18
+- Maximum: 1.20
+
+## Supported sidecar containers versions
+
+- csi-provisioner - v2.1.0
+- csi-attacher - v3.1.0
+- csi-resizer - v1.1.0
+- livenessprob - v2.2.0
+- csi-node-driver-registrar - v2.1.0
+
+## Known Issues
+
+### vSphere CSI Driver issues
+
+1. Migrated in-tree vSphere volume deleted by in-tree vSphere plugin remains on the CNS UI.
+    - Impact: Migrated in-tree vSphere volumes deleted by in-tree vSphere plugin remains on the CNS UI.
+    - Workaround: Admin needs to manually reconcile discrepancies in the Managed Virtual Disk Catalog. Admin needs to follow this KB article - https://kb.vmware.com/s/article/2147750
+2. Volume expansion might fail when it is called with pod creation simultaneously.
+    - Impact: Users can resize the PVC and create a pod using that PVC simultaneously. In this case, pod creation might be completed first using the PVC with original size. Volume expansion will fail because online resize is not supported in vSphere 7.0 Update1.
+    - Workaround: Wait for the PVC to reach FileVolumeResizePending condition before attaching a pod to it.
+3. Deleting PV before deleting PVC, leaves orphan volume on the datastore.
+    - Impact: Orphan volumes remain on the datastore, and admin needs to delete those volumes manually using `govc` command.
+    - Upstream issue is tracked at: https://github.com/kubernetes-csi/external-provisioner/issues/546
+    - Workaround:
+        - No workaround. User should not attempt to delete PV which is bound to PVC. User should only delete a PV if they know that the underlying volume in the storage system is gone.
+        - If user has accidentally left orphan volumes on the datastore by not following the guideline, and if user has captured the volume handles or First Class Disk IDs of deleted PVs, storage admin can help delete those volumes using `govc disk.rm <volume-handle/FCD ID>` command.
+
+### Kubernetes issues
+
+1. Filesystem resize is skipped if the original PVC is deleted when FilesystemResizePending condition is still on the PVC, but PV and its associated volume on the storage system are not deleted due to the Retain policy.
+    - Issue: https://github.com/kubernetes/kubernetes/issues/88683
```

Comment:
FYI - This issue is fixed in 1.21 release.  So when vSphere CSI driver supports 1.21 (when it is released), this issue can be removed.
https://github.com/kubernetes/kubernetes/pull/99326

Reply:
for `v2.2.0` release, max k8s support will be 1.20, so keeping this issue.
https://deploy-preview-769--kubernetes-sigs-vsphere-csi-driver.netlify.app/compatiblity_matrix.html